### ISSUE TITLE
Support multisite -> single install convert

### ISSUE THREAD

**Issue by GaryJones** [wp-cli/core-command#32]
`wp core multisite-convert` appears to convert a single site install to multisite, but there doesn't appear to be a command that will facilitate converting a multisite back to a single-site install.

**Comment by schlessera** [comment]
`wp core multisite-convert` basically automates the process that is documented here: https://codex.wordpress.org/Create_A_Network

For the opposite conversion, there are no existing mechanisms built into WordPress, and it is a lossy process, as soon as your multisite network contains more than one site.

Two options I could think of of how to deal with this:

A. Convert main site back to single site, and just drop all subsites.
B. Clone an individual subsite as a fresh single site.

Option A. would not often be useful, and B. is more about exporting and reimporting a site, than it is a conversion.

How do you think such a multisite => single site conversion would make sense?

**Comment by GaryJones** [comment]
For my case, I've got a couple of multisite installs, where I either never created secondary sites, or I did but later archived them and just want the main site. Option A would be fine.

Others may have done a multisite install with a development/staging site as blog ID 0 and the main site as blog ID 1. So any function should have the ability (and sufficient AYS checks) to choose which site to keep.

Agreed that Option B is a different use case. But Option A, but allowing for a choice of which site to keep, would be the functionality I'd like to see.
